### PR TITLE
[oneDNN] Fix to some benchmark tests when GPU is not available

### PR DESCRIPTION
**Pull request by mdfaijul**
Currently, the following test `//tensorflow/core/kernels:nn_ops_test_cpu` has GPU related tests that need to be skipped. This PR fixes two issues on CPU with the current benchmark test:

1. A plain `return` statement in the benchmark function does not truly return, rather the system remains unresponsive.  So `state.SkipWithError(..)` is added in this PR for truely skipping the test when GPU is not available.

2. Some of the benchmark tests missed the skipping code when GPU is not available. This PR fixes those cases.

### CODE REVIEW

**Comment by mdfaijul** [comment]
@chsigg Any update on this PR review?

**Comment by mdfaijul** [comment]
@gbaned @chsigg Any update on this PR review?

File: tensorflow/core/kernels/nn_ops_test.cc [hunk 1]
```diff
@@ -112,6 +112,9 @@ static void BM_ConvFloat(::testing::benchmark::State& state, int batch,
   if (!IsGoogleCudaEnabled() && use_gpu) {
     state.SetLabel(
```
Comment (cantonios) [inline tensorflow/core/kernels/nn_ops_test.cc:113]: This label now no longer has any affect.  Can we remove these?

**Review by cantonios — CHANGES_REQUESTED** [review]
Nice catch.

I can confirm that the benchmark seems to just hang with the existing return statement.

File: tensorflow/core/kernels/nn_ops_test.cc (hunk 1 above)
Comment (mdfaijul) [reply tensorflow/core/kernels/nn_ops_test.cc]: @cantonios Thanks for the suggestion. I have removed those redundant statements.

**Review by cantonios — APPROVED** [review]

**Comment by mdfaijul** [comment]
@cantonios Could you help understanding the error from import/copybara. We are able to see the error.

**Pull request merged**